sound/soc/sof/core.c

```diff
@@ -350,7 +350,7 @@ static int sof_probe_continue(struct snd_sof_dev *sdev)
 	sdev->first_boot = false;
 
 	/* now register audio DSP platform driver and dai */
-	ret = snd_soc_register_component(sdev->dev, &sdev->plat_drv,
```

Comment:
commit message still riddled with spelling mistakes :-(

Reply:
@plbossart hmm I knew it wasnt a good idea to just stop wearing glasses. time to make that trip to the optometrist soon. fixed the typos now.